Lib/http/cookiejar.py

```diff
@@ -1169,11 +1169,14 @@ def domain_return_ok(self, domain, request):
         # Liberal check of.  This is here as an optimization to avoid
         # having to load lots of MSIE cookie files unless necessary.
         req_host, erhn = eff_request_host(request)
+        suffix_check_domain = domain
         if not req_host.startswith("."):
             req_host = "."+req_host
         if not erhn.startswith("."):
             erhn = "."+erhn
-        if not (req_host.endswith(domain) or erhn.endswith(domain)):
+        if suffix_check_domain and not suffix_check_domain.startswith("."):
+            suffix_check_domain = "." + suffix_check_domain
+        if not (req_host.endswith(suffix_check_domain) or erhn.endswith(suffix_check_domain)):
```

Comment:
New code should conform to PEP 8.

Reply:
Maybe use shorter name, e.g. `dotdomain`?

Reply:
`dotdomain` sounds good to me. Perhaps restructure the clause as below removing the assignment at the start?

```python
if domain and not domain.startswith("."):
    dotdomain = "." + domain
else:
    dotdomain = domain
```

Reply:
LGTM.

Reply:
Changed the code to use `dotdomain` as mentioned in https://github.com/python/cpython/pull/10258#discussion_r243808847. Thanks.